File: An2/MPP/mpp-proiect-csharp-DeepSmeag/networking/dto/ListInscrieriProbaDTO.cs
using mpp_proiect_csharp_DeepSmeag.models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace networking.dto
{
    [Serializable]
    public class ListInscrieriProbaDTO
    {
        private List<InscrieriProba> inscrieriProbe;

        public ListInscrieriProbaDTO(List<InscrieriProba> inscrieriProbe)
        {
            this.inscrieriProbe = inscrieriProbe;
        }

        public List<InscrieriProba> getInscrieriProbe()
        {
            return inscrieriProbe;
        }
    }
}

File: An2/MPP/mpp-proiect-java-DeepSmeag/networkingModule/src/main/java/contests/network/dto/ListInscrieriProbaDTO.java
package contests.network.dto;

import contests.model.InscrieriProba;

import java.io.Serializable;
import java.util.List;

public class ListInscrieriProbaDTO implements Serializable {
    private List<InscrieriProba> inscrieriProbe;

    public ListInscrieriProbaDTO(List<InscrieriProba> inscrieriProbe) {
        this.inscrieriProbe = inscrieriProbe;
    }

    public List<InscrieriProba> getInscrieriProbe() {
        return inscrieriProbe;
    }
}
